src/esn.calendar.libs/app/components/event/form/event-form.directive.spec.js

```diff
@@ -76,4 +78,11 @@ describe('The cal-event-form Angular module directives', function() {
 
     expect(event.defaultPrevented).to.be.false;
   });
+
+  it('should call dragModal function when hover over modal', function() {
```

Comment:
The test is not good I think, it does not test: `$('.modal-dialog').draggable({`